packages/edge/README.md

```diff
@@ -31,6 +31,65 @@ Install the Adobe Experience Platform Edge Network extension in your mobile prop
 Then follow the same document for registering the Edge extension with the Mobile Core.
 Note that initializing the SDK should be done in native code, additional documentation on how to initialize the SDK can be found [here](https://github.com/adobe/aepsdk-react-native#initializing).
 
+
+Initializing Example:
+
+iOS
+```objc
+// AppDelegate.h
+@import AEPCore;
+@import AEPEdge;
+@import AEPEdgeIdentity;
+...
+@implementation AppDelegate
+
+// AppDelegate.m
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    [AEPMobileCore registerExtensions:@[AEPMobileEdgeIdentity.class, 
+                                        AEPMobileEdge.class] completion:^{
+     ...	
+
+   }];
+   [AEPMobileCore configureWithAppId:@"yourAppID"];   
```

Comment:
Let's move this inside the completion block for registerExtensions.
Also, do we need to add the iOS setWrapperType example too?

Reply:
Add  setWrapperType to iOS.
Do we need lifecycle in the initializing example?

Reply:
No, that is covered in the core examples.